lib/lib_rss.php

```diff
@@ -845,7 +845,7 @@ function errorMessageInfo($errorTitle, $error = '') {
 	$details = '';
 	// Prevent empty tags by checking if error isn not empty first
 	if ($error) {
-		$error = htmlspecialchars($error, ENT_NOQUOTES, 'UTF-8');
+		$error = htmlspecialchars($error, ENT_NOQUOTES, 'UTF-8') . "\n";
```

Comment:
The `list()` syntax below assumes at least two lines, which is not always the case.